source/openapi-admin-v3.yaml

```diff
@@ -2232,6 +2232,46 @@ paths:
           description: Deleted
         "404":
           description: No IP addresses or CIDR blocks to delete
+  "/groups/{groupId}/apps/{appId}/security/refresh_token_expiration":
+    parameters:
+      - "$ref": "#/components/parameters/GroupId"
+      - "$ref": "#/components/parameters/AppId"
+    get:
+      tags:
+        - security
+      operationId: adminGetRefreshTokenExpiration
+      summary: Get User Refresh Token Expiration Time
+      descripti:on: |-
+        Get the current expiration time in seconds for user session refresh tokens.
+      responses:
+        "200":
+          description: Ok
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/RefreshTokenExpiration"
+        "404":
+          description: Group or App Not Found
+    put:
+      tags:
+        - security
+      operationId: adminSetRefreshTokenExpiration
+      summary: Set User Refresh Token Expiration Time
+      descripti:on: |-
+        Set the expiration time in seconds for user session refresh tokens.
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/RefreshTokenExpiration"
+      responses:
+        "204":
+          description: Expiration time updated
```

Comment:
just checking, there's no object in the response here?

Reply:
Nope - HTTP 204 means "No Content"